models/business-unit/src/division/types.ts

```diff
@@ -21,7 +20,6 @@ export type TDivisionGraphql = Omit<
   'topLevelUnit' | 'parentUnit'
 > & {
   ancestors: Array<TCompanyGraphql | TDivisionGraphql>;
-  inheritedStores: Array<Store> | null;
```

Comment:
I removed this type from here since it's already defined in Company type from the platform SDK